src/rasenmaeher_api/web/api/enrollment/views.py

```diff
@@ -1,8 +1,10 @@
 """Enrollment API views."""
+from sre_constants import SUCCESS
```

Comment:
Group imports, stdlib ones on top (preferably starting with "from typing" -imports), then empty line then 3rd party library imports then empty line and package internal imports (using relative imports)